pkg/kubelet/eviction/helpers_test.go

```diff
@@ -105,9 +114,25 @@ func TestParseThresholdConfig(t *testing.T) {
 			expectErr:               true,
 			expectThresholds:        []Threshold{},
 		},
+		"neg-reclaim": {
+			evictionHard:            "",
+			evictionSoft:            "",
+			evictionSoftGracePeriod: "",
+			evictionMinReclaim:      "memory.available=-300Mi",
+			expectErr:               true,
+			expectThresholds:        []Threshold{},
+		},
+		"duplicate-reclaim": {
+			evictionHard:            "",
+			evictionSoft:            "",
+			evictionSoftGracePeriod: "",
+			evictionMinReclaim:      "memory.available=-300Mi,memory.available=-100Mi",
```

Comment:
Actually, the all flag values test does already:
https://github.com/kubernetes/kubernetes/pull/29400/files#diff-d1674863b3df010b8b90a347f6bec9b2R60